src/fromTriangles.js

```diff
@@ -0,0 +1,44 @@
+import { inverse } from './inverse'
+import { transform } from './transform'
+import { roundElements } from './roundElements'
+
+/**
+ * Returns a matrix that transforms a triangle t1 into another triangle t2, or throws an exception if it is impossible.
+ * @param t1 [{x: number, y: number}}, {{x: number, y: number}}, {{x: number, y: number}}] array containing the three points for the triangle as a Point array
```

Comment:
This line is preventing the continuous integration build. Before to commit I suggest you to execute the following command:
```js
yarn run test:ci
```

Reply:
Here there's the link to the ci https://travis-ci.org/chrvadala/transformation-matrix/builds/489695085

Reply:
Done! Thanks :)